src/antq/util/maven.clj

```diff
@@ -79,11 +79,24 @@
      :artifact artifact
      :remote-repos remote-repos}))
 
-(defn ^Model read-pom
+(defn- ^Model read-pom*
   [^String url]
   (with-open [reader (io/reader url)]
     (.read (MavenXpp3Reader.) reader)))
 
+(defn ^Model read-pom
+  [^String url]
+  (loop []
+    (or (try
+          (read-pom* url)
+          (catch java.net.ConnectException e
+            (if (= "Operation timed out" (ex-message e))
```

Comment:
Like src/antq/util/git.clj, if there is a specific code (not message text), we should use that.
If not, we can leave it as it is.

Reply:
`java.net.ConnectException` had nothing on it when I caught it in the REPL. I expected a useful subclass, and none exist.